devices.js

```diff
@@ -1700,6 +1700,15 @@ const devices = [
     },
 
     // Nue, 3A
+    {
+        zigbeeModel: ['FB56+ZSC05HG1.0'],
+        model: 'HGZB-042',
```

Comment:
Does it have a different model? it's the same as on https://github.com/Koenkk/zigbee-shepherd-converters/blob/a981112da92ca8c9d422bccac2f8052824d7cd47/devices.js#L1714

Reply:
My bad.. sorry. It should be 'HGZB-04D' .